docs/how-to-guides/configure-dicom-server-settings.md

```diff
@@ -49,16 +49,42 @@ If you would like to update your SQL Database tier:
 ![Configure SQL2](../images/configure-sql-2.png)
 1. Click **Apply**.
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.
 
+### Application Insights
+
+By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting:

```

Comment:
Last time as I talked with Brandon , I learned that although appinsights is not enabled, our application is still using AppInsights for logging -- which is a little bit surprise to me. 
![image](https://user-images.githubusercontent.com/67345894/94598067-434b3a00-0243-11eb-9f9d-3d6256b31fb2.png)


I'm not sure how we want to phrase this to customer, also not sure how much appinsights functionalities we are utilizing, could you check with Brandon who knows this much better to figure this out?

Reply:
When you use the template to deploy, we default to deploy application insights. We wire this up using the following commands which wire up server side telemetry and our logger: 
https://github.com/microsoft/dicom-server/blob/b4240ae765d350f359ed34032a50cf73fbb70188/src/Microsoft.Health.Dicom.Web/Startup.cs#L57-L58

More information about this approach can be found about this approach [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core) and [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/ilogger)

Reply:
@brandonpollett and @pengchen0692 I updated the text, let me know what you think!